dear_petition/petition/tests/test_models.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+
+from .factories import BatchFactory, record_data
+
+from ..models import CIPRSRecord
+
+from ..utils import dt_obj_to_date
+
+pytestmark = pytest.mark.django_db
+
+
+def test_ciprs_record_create():
+    """Test that the create_record queryset method
+
+  This method should extract General, Case, and Defendant
```

Comment:
Minor fix: indent this text to another 2 spaces, so it's 4 spaces intented

Reply:
done